ebean-core/src/main/java/io/ebeaninternal/server/core/DefaultContainer.java

```diff
@@ -150,8 +154,6 @@ private void startServer(boolean online, DefaultServer server) {
         clusterManager.registerServer(server);
```

Comment:
I don't think we should register with clusterManager until the server has started.